src/styles/index.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import './styles_main/variables';
+@import './styles_main/page';
+@import './styles_header/header';
+@import './styles_main/nav';
+@import './styles_main/card';
+@import './styles_main/stars';
```

Comment:
Add to PR other `scss` files
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/39c927c5-a1a0-4164-b781-7442d2be3b76)

Reply:
"Add to PR other scss files" Like way with src? what is PR?

Reply:
> "Add to PR other scss files" Like way with src? what is PR?

Yes, you forgot to push other files. PR - pull request